2025-02-13 07:57:42,272 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-13 07:57:42,272 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-13 08:54:28,051 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-13 08:54:28,052 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-13 09:00:41,890 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-13 09:00:41,890 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-13 09:04:35,330 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-13 09:04:35,330 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-13 09:30:48,376 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-13 09:30:48,377 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-13 09:38:27,293 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-13 09:38:27,293 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-13 09:42:17,970 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-13 09:42:17,970 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-13 09:49:03,740 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-13 09:49:03,741 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-13 10:15:10,466 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-13 10:15:10,466 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-13 14:54:33,256 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-13 14:54:33,257 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-18 08:27:40,691 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-18 08:27:40,692 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-18 08:56:36,175 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-18 08:56:36,176 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-18 08:59:42,595 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-18 08:59:42,595 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-18 10:10:49,049 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-18 10:10:49,049 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-18 10:12:00,826 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-18 10:12:00,827 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-18 10:20:27,997 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-18 10:20:27,997 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-18 10:23:27,509 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-18 10:23:27,510 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-18 10:24:56,881 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-18 10:24:56,885 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-18 10:29:53,706 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-18 10:29:53,706 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-18 10:34:26,268 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-18 10:34:26,269 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-18 10:38:44,592 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-18 10:38:44,592 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-18 10:40:10,458 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-18 10:40:10,459 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-18 10:41:59,040 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-18 10:41:59,041 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-20 08:51:03,139 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-20 08:51:03,143 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
2025-02-20 08:58:51,683 - controllers.datasets_router - INFO - [datasets_router.py:49] - Attempting to list datasets in: /app/data
2025-02-20 08:58:51,686 - controllers.datasets_router - INFO - [datasets_router.py:87] - Found 5 datasets
